packages/cli/src/index.js

```diff
@@ -18,19 +16,12 @@ if (
 }
 
 // Safe to load async-based funcs
-const commands = require('./commands');
-
-// installHooks();
-notifier({ pkg }).notify();
-
-process.on('unhandledRejection', err => {
-	error(err.stack || err.message);
-});
+const { build, info, watch } = require('./commands');
 
 const prog = sade('preact').version(pkg.version);
 
 prog
-	.command('build [src]')
+	.command('build')
```

Comment:
I think it's better to push users towards the flag; avoids conflicts and (potentially) obscure errors.